api/client/volume.go

```diff
@@ -127,7 +127,13 @@ func (cli *DockerCli) CmdVolumeInspect(args ...string) error {
 	for _, name := range cmd.Args() {
 		resp, err := cli.call("GET", "/volumes/"+name, nil, nil)
 		if err != nil {
-			return err
+			if resp.statusCode== 404 {
+				fmt.Fprintf(cli.err, "404 Not Found: %s\n", name)
```

Comment:
can we use the previous message, it's more self explanatory than this `404`: `volume not found: %s\n`.